docs/releasenotes/11.0.0.rst

```diff
@@ -69,10 +69,10 @@ Specific WebP Feature Checks
 API Changes
 ===========
 
-TODO
-^^^^
+TiffImagePlugin IFD_LEGACY_API
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-TODO
+An unused setting, ``TiffImagePlugin.IFD_LEGACY_API``, has been removed.
```

Comment:
Let's move this under "Backwards Incompatible Changes"

Reply:
Done